src/views/Timesheets/components/SavedShiftsList/index.jsx

```diff
@@ -175,31 +174,31 @@ export default class SavedShiftsList extends Component {
     );
 
     let header = (
-      <Grid item xs={12} style={styles2.headerStyle}>
+      <Grid item xs={12} className={styles._headerStyle}>
         <div>
           <Grid container direction="row">
             <Grid item xs={3}>
-              <Typography className="disable_select" variant="body2" style={styles2.headerItem}>
+              <Typography className={'disable_select' + styles.headerItem} variant="body2">
```

Comment:
Again here, what was the class disable select, and is this now referencing a class called 'disable_selectheaderItem?  This won't reference both classes, I think it will try to concatenate their names.

And ditto for the other places in this file you do the same